gnovm/tests/file.go

```diff
@@ -251,7 +251,22 @@ func RunFileTest(rootDir string, path string, opts ...RunFileTestOption) error {
 					default:
 						errstr = strings.TrimSpace(fmt.Sprintf("%v", pnc))
 					}
-					if errstr != errWanted {
+
+					// Matching on long panic strings from the preprocessor can be hard/annoying. This allows support for a
+					// wildcard match for the entire string, prefix, suffix, or both.
+					matches := errstr == errWanted
+					if strings.Contains(errWanted, "*") {
+						if errWanted == "*" {
+							matches = true
+						} else if len(errWanted) > 1 && errWanted[0] == '*' && errWanted[len(errWanted)-1] == '*' {
```

Comment:
not fully get this. say where is this kind of errWanted from: (// `*`invalid operation: r<VPBlock(1,0)> (variable of type []int) is not an interface`*`)? is it manually pre-written before the test?

What I always do is to leave errWanted empty, and run the test to fill that part, (the mainly the logic in the else clause of if errWanted != "")

Reply:
Your comment makes me think of the "golden test" concept we have where you can run it to make the expected result equal to the run result. I'll check it out. In any case, this is normally set in the comment below the file test. It's just a pain to set the very long error string for preprocessor errors. Kind of like "I expect the error to contain this and I don't care about the rest."

Reply:
yes it's more about personal taste of mine, that don't want to set it manually to save time(maybe) and avoid potential mistake. 
So, it's not a problem here to me. except `*`...`*` is kinda wired, is it? haha.

Reply:
idk what i was thinking; i reverted it